11-ras-vec.Rmd

```diff
@@ -14,16 +14,16 @@ library(spDataLarge)
 
 ## Introduction
 
-In this section we will focus on the interactions between a raster and vector model.
-It includes four main techniques.
-Raster cropping and masking using vector objects is covered in section \@ref(raster-cropping).
-In section \@ref(raster-extraction), we will describe how raster values can be extract using different types of vector data - points, lines and polygons.
-Finally, sections \@ref(rasterization) and \@ref(spatial-vectorization) show how to convert vectors into rasters and raster images into vectors.
+This section focusses on interactions between raster and data vector geographic data models, introduced in Chapter \@ref(spatial-class).
```

Comment:
Note: cross referencing previous chapter for cohesion and referring to it as a section still as it will eventually be one.